src/v2/Apps/Partner/Components/PartnerContacts/PartnerContactMap.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from "react"
+import styled from "styled-components"
+import { Link, Image, ResponsiveBox } from "@artsy/palette"
+import {
+  getGoogleMapUrl,
+  getGoogleStaticMapImageUrl,
+} from "./partnerContactUtils"
+import { PartnerContactCard_location } from "v2/__generated__/PartnerContactCard_location.graphql"
+import { useWindowSize } from "v2/Utils/Hooks/useWindowSize"
+import { unitlessBreakpoints } from "@artsy/palette/dist/themes/v3"
```

Comment:
So this is a problem in that these breakpoints don't match up exactly. In general: don't import theme values (there are now 2 themes).

Rather; use our Responsive component and render both images.

Reply:
I found `Responsive` [component](https://github.com/artsy/force/blob/352e61c0c3ad1f7de3b13ecdc6bd233ac43b5441/src/v2/Utils/Responsive/DeprecatedResponsive.tsx#L14) but it looks deprecated, I used `Media` component is it okay?